CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### CHANGED
+
+- Upgraded for usage with terraform-0.12.x [#3]
```

Comment:
The URL for the #3 link must be added to the bottom of the file, like:

```
[#3]: https://github.com/terraform-google-modules/terraform-google-folders/pull/3
```

Reply:
Ok, got it

Reply:
Thank you!

Reply:
This is also a wrong issue number. Change to "#2"